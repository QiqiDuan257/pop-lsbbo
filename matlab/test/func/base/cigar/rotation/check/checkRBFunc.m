clear; clc;

% nohup matlab -nodesktop -nodisplay -nosplash <checkRBFunc.m >checkRBFunc.out 2>&1 &

funcName = 'popRFuncCigar';

%%
popCheckCommRBFunc(funcName);
