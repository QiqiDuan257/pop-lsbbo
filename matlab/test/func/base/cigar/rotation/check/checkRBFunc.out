
                            < M A T L A B (R) >
                  Copyright 1984-2016 The MathWorks, Inc.
                   R2016b (9.1.0.441655) 64-bit (glnxa64)
                             September 7, 2016

#############################################################################
#             $$$ Enjoy the <pop-lsbbo> pure-MATLAB Library $$$             #
#                        --- version: 0.00.000.1 ---                        #
#############################################################################
popRFuncCigar: pass the test.
     3: runtime 1.65e-01.
popRFuncCigar: pass the test.
     4: runtime 1.57e-02.
popRFuncCigar: pass the test.
     5: runtime 4.45e-03.
popRFuncCigar: pass the test.
     6: runtime 4.30e-03.
popRFuncCigar: pass the test.
     7: runtime 1.30e-02.
popRFuncCigar: pass the test.
     2: runtime 4.02e-03.
popRFuncCigar: pass the test.
    10: runtime 3.55e-03.
popRFuncCigar: pass the test.
   100: runtime 5.37e-02.
popRFuncCigar: pass the test.
  1000: runtime 4.29e+00.
popRFuncCigar: pass the test.
  2000: runtime 1.74e+01.
popRFuncCigar: pass the test.
  4000: runtime 6.88e+01.
popRFuncCigar: pass the test.
  6000: runtime 1.54e+02.
popRFuncCigar: pass the test.
  8000: runtime 2.74e+02.
popRFuncCigar: pass the test.
 10000: runtime 4.28e+02.
Total runtime: 9.46999e+02.
